debugdrawer/src/main/kotlin/au/com/gridstone/debugdrawer/DebugDrawer.kt

```
@@ -48,6 +49,22 @@ object DebugDrawer {
    */
   fun with(activity: Activity): Builder = Builder(activity)
 
+  /**
+   * Closes the debug drawer if one has been installed in the activity.
+   */
+  fun closeIn(activity: Activity) {
+    val layout: DrawerLayout? = activity.findViewById(R.id.debugDrawerId)
+    layout?.closeDrawer(GravityCompat.END)
```

Not a fan of the silent fail here though

---

You reckon this method should through an `IllegalStateException`  if you attempt to close the drawer when there isn't one?

---

a throw would be pretty extreme - perhaps a consumable `Boolean` back?